dags/openshift_nightlies/config/install/hypershift/node-type.json

```diff
@@ -0,0 +1,16 @@
+{
+    "cluster_install_method": "osd",
```

Comment:
Why do we need this node-type.json?

Reply:
good catch, typo. should be `none-type.json` for no compute hosted cluster